### DISCUSSION TITLE
When using input (), I can input multiple lines like IPython

### DISCUSSION THREAD

**Discussion by ydmykr** [microsoft/vscode-jupyter#11059] [New Ideas]
For example, when I run the code : 
![image](https://user-images.githubusercontent.com/100815370/183001871-138b170f-e7a4-480a-af58-4fdc5cb59728.png)

I'll get three input boxes.

But if it's in .py or ipython I can use `"ctrl+enter`" or `"ctrl+o"` to input multiple lines.

![image](https://user-images.githubusercontent.com/100815370/183002007-2c3f61b0-ce6d-4663-9f4f-3b30abb62369.png)

I refer to this [issue](https://github.com/ipython/ipython/issues/2493) in GitHub.

I hope that in the input box of jupyter notebook, when I use a shortcut key such as `"ctrl+enter"`, my input box can be enlarged and line feed input (I can see the content entered previously) and reduce the number of required inputs (for example, I need to input three times for the code in the picture, and the number of input boxes can be changed from three times to two times after I use the shortcut key to line feed)

**Comment by DonJayamanne** [comment]
@ydmykr from what I can tell this is the same behaviour in Jupyter Lab & Jupyter Notebooks as well, isn't that correct.
